x-pack/plugins/fleet/server/saved_objects/migrations/security_solution/to_v8_7_0.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { SavedObjectMigrationFn, SavedObjectUnsanitizedDoc } from '@kbn/core/server';
+import { cloneDeep } from 'lodash';
+
+import type { PackagePolicy } from '../../../../common';
+
+export const migratePackagePolicyToV870: SavedObjectMigrationFn<PackagePolicy, PackagePolicy> = (
+  packagePolicyDoc
+) => {
+  if (packagePolicyDoc.attributes.package?.name !== 'endpoint') {
+    return packagePolicyDoc;
+  }
+
+  const updatedPackagePolicyDoc: SavedObjectUnsanitizedDoc<PackagePolicy> =
+    cloneDeep(packagePolicyDoc);
```

Comment:
`cloneDeep` is expensive. Is there any reason why this would be mandatory here? Looking at the previous migrations, it seems the operation wasn't done

e.g https://github.com/elastic/kibana/blob/3fa88f289609ae8565e9d6b05d502adcc912b43a/x-pack/plugins/fleet/server/saved_objects/migrations/to_v8_6_0.ts#L43

Reply:
You are right that it isn't necessary to use `cloneDeep` here. This was a mindless copy pasta on my end since all of our previous endpoint migrations, such as in `migrations/security_solution/to_v8_6_0.ts`, have used it. I'll make it a normal shallow copy, good catch!